vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -764,103 +765,158 @@ private void inviteNoTchapContactsByEmail (final ArrayList<String> emails, final
         mCount = emails.size();
         mSuccessCount = 0;
 
-        for (final String email : emails) {
-            // We check if this email has been already invited
-            // (pendingInvites are ignored here because we could not have a pending invite related to an email)
-            Room existingRoom = DinsicUtils.isDirectChatRoomAlreadyExist(email, mSession, false);
+        final List<String> medias = new ArrayList<>();
 
-            if (null != existingRoom) {
-                // If a direct chat already exists, we do not re-invite the NoTchapUser
-                // and we notify the user by a toast
-                String message = getString(R.string.tchap_invite_already_send_message, email);
+        for (String email : emails) {
+            medias.add(ThreePid.MEDIUM_EMAIL);
+        }
+
+        // Check for each email whether there is an associated account with a Matrix id.
```

Comment:
This check must not be done here. 
It is expected only when the user types severals emails in the dialog. 
You must use lookup3Pids at line 707 to remove unexpected emails from the provided list before calling "inviteNoTchapContactsByEmail"